### ISSUE TITLE
"Open Hasura" always uses 9695

### ISSUE THREAD

**Issue by mainstreetmark** [nhost/cli#486]
Starting a local nhost via

```
/usr/local/bin/nhost dev --ui nhost --port 9009 --console-port 9002 
```

The resulting Dashboard has that Hasura tool.  But when clicking on "Open Hasura", it opens to `localhost:9695` instead of 9002.

Similarly, the Database tool errors on "Cannot read properties of null (reading 'awsName')", which seems like it doesn't know which port the database is running on.  Perhaps it is also using the original port, rather than the one specified on the CLI?

**Comment by elitan** [comment]
> The resulting Dashboard has that Hasura tool. 

What do you mean with this?

> But when clicking on "Open Hasura", it opens to localhost:9695 instead of 9002.
WHere do you click? What's the output of the terminal (`nhost dev`)?

> Similarly, the Database tool errors on "Cannot read properties of null (reading 'awsName')",
What Database tool?

> which seems like it doesn't know which port the database is running on. Perhaps it is also using the original port, rather than the one specified on the CLI?

Which service and port do you refer to? You're using `--port 9009` in the command but I don't think we have a `--port` flag for our CLI.

**Comment by mainstreetmark** [comment]
So, I run that command, and it spins up nhost, and opens the "Dashboard".  The Dashboard has a Hasura button on the left, and clicking it opens the "Open Hasura" page.  That page shows the Admin Secret and has a "Open Hasura" button.  Clicking the "Open Hasura" button opens http://localhost:9695/console.

The result of the `nhost` command shows

   - Hasura Console:       http://localhost:9002

So, I would expect the "open Hasura" button to open http://localhost:9002, instead of http://localhost:9695

**Comment by szilarddoro** [comment]
Hello @mainstreetmark! Are you using the latest CLI version? This shouldn't be a problem in recent versions.

**Comment by elephant3** [comment]
Should be fixed in the latest v0.9.1, please upgrade
